recipes/libsndfile/all/conanfile.py

```diff
@@ -53,6 +56,9 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+        if self.options.with_sqlite != "deprecated":
+            raise ConanInvalidConfiguration("with_sqlite is a deprecated option. Do not use.")
```

Comment:
Should this be a warning instead?
Or is an error ok?
```suggestion
            self.output.warn("with_sqlite is a deprecated option. Do not use.")
```

Reply:
I don't think that an error makes sense. It breaks the build equally as when we just remove the option completely already. So I would go with a warning